### PR TITLE
New Catalog item: Test Containers Demo

### PR DESCRIPTION
**Pull request by deewhyweb**
##### ISSUE TYPE

- New role Pull Request

##### COMPONENT NAME
Test containers Demo

##### ADDITIONAL INFORMATION

```New demo catalog item to install and configure Docker Test Containers on OpenShift
```

### CODE REVIEW

**Review by bbethell-1 — APPROVED** [review]

File: ansible/roles/ocp4-workload-test-containers/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+galaxy_info:
+  role_name: ocp4-workload-test-containers
+  author: 
+    - Application Service BU Runtimes Team, Philip Hayes (phayes@redhat.com)
+    - Application Service BU Runtimes Team, Daniel Oh (doh@redhat.com)
+  description: |
+    Create Docker Test Containers on OpenShift 4 cluster.
+    This catalog entry is developed and maintained by the Application Services BU.
```
Comment (danieloh30) [inline ansible/roles/ocp4-workload-test-containers/meta/main.yml:9]: shouldn't it be AppDeveloper BU?

**Review by juliaaano — CHANGES_REQUESTED** [review]
Please consider the following changes:

1. Assuming this workload runs on OpenShift, please move it to the `roles_oct_workloads` folder.
2. Use underscore instead of dashes for the role name. I suggest this name: `ocp4_workload_testcontainers`.
3. We've got workloads to install kubevirt and pipelines. Could you use them on your agnosticV config instead of installing on your own (ocp4_workload_kubevirt, ocp4_workload_pipelines)?
4. User your own variables in defaults/main.yaml, with the prefix of the role name. For example, ocp4_workload_testcontainers_num_users.
5. The other defaults/main.yml prefixed with workshop are not used, I noticed. Please remove them.
6. Finally, please consider extracting some vars, such as tcc-operator CSV, using the naming convention described above.

File: ansible/roles/ocp4-workload-test-containers/meta/main.yml (hunk 1 above)
Comment (deewhyweb) [reply ansible/roles/ocp4-workload-test-containers/meta/main.yml:9]: Thanks Dan!

**Comment by deewhyweb** [comment]
@juliaaano I've made those changes, please review again.

Thanks,
Philip

**Review by juliaaano — APPROVED** [review]

**Pull request merged**